api/src/main/java/openconsensus/internal/Utils.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2019, OpenConsensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package openconsensus.internal;
+
+/** General internal utility methods. */
+public final class Utils {
```

Comment:
nit: could move many methods to a class called `Check`, e.g. `Check.argument()`, `Check.notNull()`.